Python/Password Checker/README.md

```diff
@@ -0,0 +1,20 @@
+# Password_Checker
+
+This self explanatory script, uses [PawnedPassword_API](https://pawnedpasswords.com).
+It checks how many times your password has been under hacaker radars.
+It's pretty cool.
```

Comment:
Don't break your statement too often. Use full statements. And we definitely don't need "It's pretty cool" on the README. Follow the standard mentioned [here](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md) and stick to the point.